MoreMimics/Mods/MoreMimics/ScriptExtender/Lua/BootstrapServer.lua

```diff
@@ -4,10 +4,55 @@ local utils = Ext.Require("utils.lua")
 
 ModuleUUID = "c19ca43a-c3c7-4e58-9d00-f7d928e72074"
 
+PersistentVars = {}
```

Comment:
i Usew it to transfer "ChestQuality" to the mimic. I havn't find original chestId in the mimic. So i use that to make map of mimicId -> originalChestQuality

may be have another way to transfer it.  Or may be on mimic kill, remove his id from the map

Reply:
I think you give it a custom status for their quality when you spawn them then check for them instead using Osi.HasActiveStatus

new entry "CHEST_QUALITY_POOR"
type "StatusData"
data "StatusType" "BOOST"
data "StatusPropertyFlags" "DisableOverhead;DisableCombatlog;DisablePortraitIndicator"

Reply:
Status make icône and visible for user on détail. Not sure it's good for that case

Reply:
There are ways to hide it from user, DisableOverhead;DisableCombatlog;DisablePortraitIndicator <- should make it not show to the user.

Also in the future we might want to have an indicator of its quality anyway like if it's good the mimic would have a golden effect like this statuseffect

`data "StatusEffect" "d56b562b-f434-4521-b11e-b4e5083af370"`

Reply:
Also for difficulty... yes there should be a way of scaling the mimics damage. I was thinking of adding 2 new mimics

EasyMimic, Mimic (what is used in game already) and HardMimic

EasyMimic spawns at lvl 1-5 (Act 1), Mimic for 6-10 (Most of Act 2), HardMimic for 11-12 (Act 3)

We'll need a new name for "HarderMimics" in the configs, maybe instead it should be called "Equipment Stealing Power" or something.

Only problem is when I added the new easy mimic the save no longer works when you uninstall the mod but I think there's no avoiding that if we add new monsters

Reply:
i don't kown how to do mech but make wardrobe mimics would be very cool :D, i have all ID for wardrobe in 3 category too

Reply:
"Equipment Stealing Power" could be a think to separate that power

an other one with mimic power: 1 to A level , could be nice  to have something like that : 
1; basic 1 attack
..
Max level : somothing like character lvl +2,  1 extra attack, and  2 random immunities + 4 random resistances 

as i tell u. i haven't try Combat extender if mimics go t oplayer level if configured for (Player Level sclaing to work one the created mimic). If it work. may be take care of interactions

Reply:
> EasyMimic spawns at lvl 1-5 (Act 1), Mimic for 6-10 (Most of Act 2), HardMimic for 11-12 (Act 3)

take care of those having lvl 20 enabler

Reply:
did i rebase it, or you dont want it ?